File: Python/omegaup.com/Matriz-de-Adyacencia.py
n, m = map(int, input().split())
gr = []
for i in range(n):
    gr.append([0]*n)

for i in range(m):
    origen, destino = map(int, input().split())
    gr[origen][destino] = 1
    gr[destino][origen] = 1

for i in range(n):
    for j in range(n):
        print(gr[i][j], end=' ')
    print()

File: cpp/omegaup.com/Matriz-de-Adyacencia.cxx
#include <iostream>

using namespace std;

int main(){
    int n, m, origen, destino;
    cin >> n >> m;
    int gr[n][n];
    for(int i=0;i<n;i++){
        for(int j=0;j<n;j++){
            gr[i][j] = 0;
        }
    }
    for(int i=0;i<m;i++){
        cin>>origen>>destino;
        gr[origen][destino] = 1;
        gr[destino][origen] = 1;
    }

    for(int i=0;i<n;i++){
        for(int j=0;j<n;j++){
            cout<<gr[i][j]<<" ";
        }
        cout<<endl;
    }

    return 0;
}